src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -747,3 +747,140 @@
+examples:
+  - name: Updates the scope map 'myScopeMap' replacing old actions with 'repositories/hello-world/push'.
+    text: >
+        az acr scope-map update -r myRegistry -n myScopeMap --add hello-wolrd,push --reset
+"""
+
+helps['acr scope-map show'] = """
+type: command
+short-summary: Shows details and attributes for a scope map under an Azure Container Registry.
+examples:
+  - name: Get information for the scope map 'myScopeMap'.
+    text: >
+        az acr scope-map show -r myRegistry -n myScopeMap
+"""
+
+helps['acr scope-map list'] = """
+type: command
+short-summary: Lists all scope maps under an Azure Container Registry.
+examples:
+  - name: List scope maps under the registry 'myRegistry'.
+    text: >
+        az acr scope-map list -r myRegistry
+"""
+
+helps['acr token'] = """
+type: group
+short-summary: Manage tokens for an Azure Container Registry.
+"""
+
+helps['acr token create'] = """
+type: command
+short-summary: Creates a token for an Azure Container Registry.
+examples:
+  - name: Create a token associated to the scope map 'myScopeMap'.
+    text: >
+        az acr token create -r myRegistry -n myToken --scope-map myScopeMap
+"""
+
+helps['acr token delete'] = """
+type: command
+short-summary: Deletes a token under an Azure Container Registry.
+examples:
+  - name: Delete the token 'myToken'.
+    text: >
+        az acr token delete -r myRegistry -n myToken
+"""
+
+helps['acr token update'] = """
+type: command
+short-summary: Updates a token under an Azure Container Registry, replacing the scope map associated to it.
+examples:
+  - name: Updates the token 'myToken', making it associated with the scope map 'myScopeMap'.
+    text: >
+        az acr token update -r myRegistry -n myToken --scope-map myScopeMap
+"""
+
+helps['acr token show'] = """
+type: command
+short-summary: Shows details and attributes for a token under an Azure Container Registry.
+examples:
+  - name: Get information for the token 'myToken'.
+    text: >
+        az acr token show -r myRegistry -n myToken
+"""
+
+helps['acr token list'] = """
+type: command
+short-summary: Lists all tokens under an Azure Container Registry.
+examples:
+  - name: List tokens under the registry 'myRegistry'.
+    text: >
+        az acr token list -r myRegistry
+"""
+
+helps['acr token credential'] = """
+type: group
+short-summary: Manage credentials entities for a token under an Azure Container Registry.
+"""
+
+helps['acr token credential generate'] = """
+type: command
+short-summary: Generates both passwords for a token under an Azure Container Registry, unless one password is specified. Replaces old values if they existed. If expiry is not specified, it will be defaulted to the max value.
+examples:
+  - name: Generates passowrd1 for the token 'myToken', with expiration for 2 years.
```

Comment:
nit: password

Reply:
@shahzzzam There are two password parameters: `--password1` and `--password2`

Reply:
Yes, by now the design suggestion is having flags for each password, but if we change it I will keep reviewing the `help` as priority. Thank you!

Reply:
Oh no, I meant the spelling of `password1` is incorrect. :) I understand why we need pw1 and pw2